<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmListaRodadasAbertas</class>
 <widget class="QDialog" name="frmListaRodadasAbertas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs/imagens.qrc">
    <normaloff>:/localizarRodada.png</normaloff>:/localizarRodada.png</iconset>
  </property>
  <widget class="QTableWidget" name="twgListaRodadasAbertas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>989</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>110</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>QRA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtQra">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>110</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Indicativo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtIndicativo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>110</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtId">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>231</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Título</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTitulo">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>231</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>TG ou Frequência</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtTgFrequencia">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>110</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Horário</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtHorario">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>101</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Edição da Rodada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEdicaoRodada">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>191</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Data Hora de Início</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtDataHoraInicio">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>140</y>
     <width>191</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Data Hora Fim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtDataHoraFim">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>991</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Comentários</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtComentarios">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnExcluirRodada">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/PessoaRemover.png</normaloff>:/PessoaRemover.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAtualizarDados">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/recarregar.png</normaloff>:/recarregar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFecharRodada">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/finalizar.png</normaloff>:/finalizar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRetornarRodada">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Retornar a Rodada"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/retornar.png</normaloff>:/retornar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Sair"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/sair.png</normaloff>:/sair.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>45</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
